doc/source/data/preprocessors.rst

```diff
@@ -5,7 +5,14 @@ Using Preprocessors
 
 Data preprocessing is a common technique for transforming raw data into features for a machine learning model.
 In general, you may want to apply the same preprocessing logic to your offline training data and online inference data.
-Ray AIR provides several common preprocessors out of the box and interfaces to define your own custom logic.
+
+This page covers *preprocessors*, which are a higher level API on top of existing Ray Data operations like `map_batches`,
+targeted towards tabular and structured data use cases.
+
+The recommended way to perform preprocessing is to :ref:`use existing Ray Data operations <transforming_data>` instead
```

Comment:
Can we update the wording here? It's a little bit confusing right now if we say the recommended way for preprocessing is existing Ray Data operations, but then say you should consider built-in preprocessors.

Maybe just "While Ray Data supports generic transformations on datasets, for tabular data it also provides built-in preprocessors"